version.hh

```diff
@@ -24,7 +24,7 @@ public:
     }
 
     static version current() {
-        static version v(3, 0, 8);
+        static version v(4, 0, 0);
```

Comment:
There's a risk here that if several different things are advertised by the version, and we only implemented one of them (server-side describe), then we'll confuse whatever uses the version to look for the other things.

If that's the case, then we'll have to revert the patch, and teach cqlsh to discover DESCRIBE in some other way (for example by running it).

Reply:
I've looked through the cqlsh code and didn't find any other references to the version (besides print at the start of cqlsh). 

But even if there were other features, I don't know if there is any other way to inform cqlsh about server-side describe, because is code there is if statement that check whether the version is version is >= 4 
https://github.com/jeffwidman/cqlsh/blob/master/cqlsh/cqlsh.py#L1372-L1376

Reply:
I'm more worried about other drivers, not cqlsh. Well, we'll find out.